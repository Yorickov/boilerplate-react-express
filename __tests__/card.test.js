import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../client/components/Card';

it('Card', () => {
  const params = {
    title: 'title',
    text: 'text',
  };
  const component = renderer.create(<Card {...params} />);
  expect(component.toJSON()).toMatchSnapshot();
});
