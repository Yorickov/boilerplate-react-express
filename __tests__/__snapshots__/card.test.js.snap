// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card 1`] = `
<div
  className="card"
>
  <div
    className="card-body"
  >
    <h4
      className="card-title"
    >
      title
    </h4>
    <p
      className="card-text"
    >
      text
    </p>
    <button
      className="btn btn-primary"
      type="button"
    >
      Go somewhere
    </button>
  </div>
</div>
`;
